core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/source/scan/csvOld/CSVOldScanSourceOpExec.scala

```diff
@@ -51,7 +53,8 @@ class CSVOldScanSourceOpExec private[csvOld] (
     implicit object CustomFormat extends DefaultCSVFormat {
       override val delimiter: Char = customDelimiter.get.charAt(0)
     }
-    reader = CSVReader.open(filePath, fileEncoding.getCharset.name())(CustomFormat)
+    val filePath = openFile(new URI(fileUri)).asFile().toPath
```

Comment:
also, are we closing file properly after usage?

Reply:
see the above reply